event-sourcing/event-store-cassandra/src/main/java/org/apache/james/eventsourcing/eventstore/cassandra/EventStoreDao.java

```diff
@@ -97,10 +97,9 @@ private BoundStatement insertEvent(Event event) {
     }
 
     public History getEventsOfAggregate(AggregateId aggregateId) {
-        return cassandraAsyncExecutor.execute(
+        return cassandraAsyncExecutor.executeRows(
```

Comment:
I think that we are wrong in this method: events should be ordered by their id before building the history. Would you mind checking we cover that ?

Reply:
It's already verified in `EventStoreTest.getEventsOfAggregateShouldReturnAppendedEvents()`